docs/HACKING.md

```diff
@@ -249,3 +249,8 @@ git submodule update --recursive
 # To update a specific submodule
 git submodule update --remote <path_to_submodule>
 ```
+
+## Preventing Windows Machines from being configured by WMCO
+
+If the Machine spec has the label `windowsmachineconfig.openshift.io/ignore=true`, the Machine will be ignored by WMCO,
+and will not be configured into a Windows node. This can be helpful when debugging userdata changes.
```

Comment:
> the Machine will be ignored by WMCO and will not be configured into a Windows node

unless it is added as an entry to the windows-instances ConfigMap, right? Since this is hack readme, might be good to call this out.

Reply:
I'm avoiding calling that out specifically because I dont want anyone to assume thats okay to do.
I'll change the language to specifically call out the Machine controller will ignore the Machine though.